python/cuml/test/test_mbsgd_classifier.py

```diff
@@ -75,6 +72,7 @@ def test_mbsgd_classifier(datatype, lrate, input_type, penalty,
 @pytest.mark.parametrize('column_info', [unit_param([20, 10]),
                          quality_param([100, 50]),
                          stress_param([1000, 500])])
+@pytest.mark.xfail(raises=AssertionError)
```

Comment:
same question as above

Reply:
This was because `eta0 (initial learning rate value) = 0`  was throwing an error. Issue #1105 has been resolved. Will find the PR for that and close the issue. Removing the xfail from both places.